activesupport/lib/active_support/core_ext/numeric/time.rb

```diff
@@ -5,6 +5,11 @@
 require 'active_support/core_ext/date/acts_like'
 
 class Numeric
+  # Convert number to local Time.
+  def to_time
+    Time.at(self)
```

Comment:
@pixeltrix I think I feel like the numeric -> time conversion is too special-case to warrant making all numerics respond to `to_time`, even though it's still an explicit conversion. :confused:

I guess my contention is that the choice of time unit & epoch are both domain concerns of `Time`, so it feels a stretch to leak them to `Numeric`. (Which I think might be more obvious once we expand this method's documentation to a workable level of detail.)